Packages/com.unity.inputsystem/InputSystem/Editor/AssetImporter/InputActionImporter.cs

```diff
@@ -117,6 +131,76 @@ private static InputActionAsset CreateFromJson(AssetImportContext context)
             return asset;
         }
 
+        internal static void MigrateAllEnumParams(InputActionAsset asset)
```

Comment:
This is mutating the initially parsed InputActionAsset. I would expect this to be inside InputActioNAsset since its part of Parsing the asset and is not only edit-mode.